### PR TITLE
[build] Polish repo.spring.io repositories, remove libs prefix

### PR DESCRIPTION
**Pull request by bsideup**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -69,10 +69,10 @@ configure(allprojects) { project ->
   group = 'io.projectreactor.kafka'
 
   repositories {
-    maven { url 'https://repo.spring.io/libs-release' }
-    maven { url 'https://repo.spring.io/libs-snapshot' }
     mavenCentral()
     jcenter()
+    maven { url 'https://repo.spring.io/libs-release' }
+    maven { url 'https://repo.spring.io/libs-snapshot' }
   }
```
Comment (violetagg) [inline build.gradle:76]: you should switch also to `https://repo.spring.io/release` and `https://repo.spring.io/snapshot`

Comment (bsideup) [reply build.gradle:76]: I kept it consistent with reactor-core:
https://github.com/reactor/reactor-core/blob/669df3dbe7b7907288fe248bf1ee84a3dfcaf3fc/build.gradle#L131

Comment (simonbasle) [reply build.gradle:76]: reactor-core is actually wrong, it is preferable to use the ones without `libs-`

Comment (simonbasle) [reply build.gradle:76]: > January 6, 2021
> 
> We will no longer support anonymous download of 3rd-party artifacts from repo.spring.io. The /libs-snapshot, /libs-milestone, and /libs-release repositories, which the Spring Team uses to build and release Spring projects, will require authentication.
The `/snapshot`, `/milestone`, and /release repositories will remain available, but please fetch our releases from a central repository.

source: https://spring.io/blog/2020/10/29/notice-of-permissions-changes-to-repo-spring-io-fall-and-winter-2020

**Review by simonbasle — APPROVED** [review]

**Review by simonbasle — APPROVED** [review]
just tested locally and the `repositories` for build script (plugins) needs also `jcenter()`

**Review by simonbasle — CHANGES_REQUESTED** [review]
just tested locally and the `repositories` for build script (plugins) needs also `jcenter()`

**Review by simonbasle — APPROVED** [review]
sorry I'm getting confused, the plugin part was missing in another project 🤦  will merge this and forward merge

**Pull request merged**